pkg/controller/scyllacluster/resource.go

```diff
@@ -882,6 +925,20 @@ func stringOrDefault(str, def string) string {
 	return def
 }
 
+func ptrValueOrDefault[T any](v *T, def T) T {
```

Comment:
please use a verb for the func name, e.g. getValueOrDefault?

Reply:
done